pkg/monitor/monitor_test.go

```diff
@@ -23,14 +23,15 @@ import (
 )
 
 const (
-	mockDriverName        = "MockDriver"
+	mockDriverName = "MockDriver"
+	nodeForPod     = "node1.domain"
+)
+
+var (
 	driverVolumeName      = "singleVolume"
```

Comment:
Had to make these variables, as `Source.PersistentVolumeName` takes string address.

Source: storagev1beta1.VolumeAttachmentSource{
				PersistentVolumeName: &attachmentVolumeName,
},